kernel/contracts/KernelStakes.sol

```diff
@@ -0,0 +1,55 @@
+pragma solidity ^0.4.18;
+
+import "./ZepToken.sol";
+import "zeppelin-solidity/contracts/math/SafeMath.sol";
+import "zeppelin-solidity/contracts/ownership/Ownable.sol";
+
+contract KernelStakes is  Ownable {
+  using SafeMath for uint256;
+
+  event Staked(address indexed user, address instance, uint256 amount, uint256 total, bytes data);
+  event Unstaked(address indexed user, address instance, uint256 amount, uint256 total, bytes data);
+
+  uint256 private _totalStaked;
+
+  mapping(address => uint256) private _instanceVouches;
+  mapping(address => mapping (address => uint256)) private _userVouches;
+
+  function KernelStakes() { }
+
+  function stake(address staker, address instance, uint256 amount, bytes data) public onlyOwner {
+    _totalStaked = _totalStaked.add(amount);
+    _instanceVouches[instance] = _instanceVouches[instance].add(amount);
+    _userVouches[staker][instance] = _userVouches[staker][instance].add(amount);
+
+    Staked(staker, instance, amount, _instanceVouches[instance], data);
+  }
+
+  function unstake(address staker, address instance, uint256 amount, bytes data) public onlyOwner {
+    uint256 currentStake = _userVouches[staker][instance];
+    require(currentStake >= amount);
+
+    _totalStaked = totalStaked().sub(amount);
+    _instanceVouches[instance] = _instanceVouches[instance].sub(amount);
+    _userVouches[staker][instance] = currentStake.sub(amount);
+    
+    Unstaked(staker, instance, amount, _instanceVouches[instance], data);
+  }
+
+  function transferStake(address staker, address from, address to, uint256 amount, bytes data) public onlyOwner {
+    unstake(staker, from, amount, data);
+    stake(staker, to, amount, data);
```

Comment:
we should send the payout to the new dev here

Reply:
Yup, I embarrasingly missed it. Pushing a fix now!

Reply:
I would remove this method altogether now, since it is no longer called from `ZepCore`'s `transferStake`...